Build is failing on travis CI because of Gosu

Gosu is failing to build on travis CI.

```
Using worker: worker-linux-2-1.bb.travis-ci.org:travis-linux-13
git.1
$ git clone --depth=50 --branch=master git://github.com/mrhead/ruby_arena.git mrhead/ruby_arena
Cloning into 'mrhead/ruby_arena'...
remote: Counting objects: 333, done.
remote: Compressing objects: 100% (164/164), done.
remote: Total 333 (delta 211), reused 278 (delta 160)
Receiving objects: 100% (333/333), 40.36 KiB, done.
Resolving deltas: 100% (211/211), done.
$ cd mrhead/ruby_arena
git.3
$ git checkout -qf df4858dced68866c129116136520e4457021205a
$ rvm use 2.1.0 --install --binary --fuzzy
Using /home/travis/.rvm/gems/ruby-2.1.0
$ export BUNDLE_GEMFILE=$PWD/Gemfile
$ ruby --version
ruby 2.1.0p0 (2013-12-25 revision 44422) [x86_64-linux]
$ rvm --version
rvm 1.24.11 (latest-minor) by Wayne E. Seguin <wayneeseguin@gmail.com>, Michal Papis <mpapis@gmail.com> [https://rvm.io/]
$ gem --version
2.2.0
$ bundle install --deployment
Fetching gem metadata from https://rubygems.org/.........
Fetching additional metadata from https://rubygems.org/..
Installing diff-lcs (1.2.5)
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.
    /home/travis/.rvm/rubies/ruby-2.1.0/bin/ruby extconf.rb 
The Gosu gem requires some libraries to be installed system-wide.
See the following site for a list:
https://github.com/jlnr/gosu/wiki/Getting-Started-on-Linux
checking for TTF_RenderUTF8_Blended() in -lSDL_ttf... no
checking for FreeImage_ConvertFromRawBits() in -lfreeimage... no
checking for main() in -lopenal... no
creating Makefile
make "DESTDIR=" clean
make "DESTDIR="
compiling RubyGosu_wrap.cxx
cc1plus: warning: command line option ‘-Wdeclaration-after-statement’ is valid for C/ObjC but not for C++ [enabled by default]
cc1plus: warning: command line option ‘-Wimplicit-function-declaration’ is valid for C/ObjC but not for C++ [enabled by default]
In file included from RubyGosu_wrap.cxx:1:0:
../GosuImpl/RubyGosu_wrap.cxx: In function ‘void SWIG_Ruby_define_class(swig_type_info*)’:
../GosuImpl/RubyGosu_wrap.cxx:1468:9: warning: variable ‘klass’ set but not used [-Wunused-but-set-variable]
In file included from RubyGosu_wrap.cxx:1:0:
../GosuImpl/RubyGosu_wrap.cxx: In function ‘VALUE SWIG_AUX_NUM2DBL(VALUE*)’:
../GosuImpl/RubyGosu_wrap.cxx:2428:9: warning: unused variable ‘type’ [-Wunused-variable]
In file included from RubyGosu_wrap.cxx:1:0:
../GosuImpl/RubyGosu_wrap.cxx: In function ‘void Gosu_Window_transform(Gosu::Window*, double, double, double, double, double, double, double, double, double, double, double, double, double, double, double, double)’:
../GosuImpl/RubyGosu_wrap.cxx:2837:9: warning: missing braces around initializer for ‘std::tr1::array<double, 16ul>::value_type [16] {aka double [16]}’ [-Wmissing-braces]
In file included from RubyGosu_wrap.cxx:1:0:
../GosuImpl/RubyGosu_wrap.cxx: In member function ‘virtual void SwigDirector_Window::update()’:
../GosuImpl/RubyGosu_wrap.cxx:2925:9: warning: variable ‘result’ set but not used [-Wunused-but-set-variable]
../GosuImpl/RubyGosu_wrap.cxx: In member function ‘virtual void SwigDirector_Window::draw()’:
../GosuImpl/RubyGosu_wrap.cxx:2932:9: warning: variable ‘result’ set but not used [-Wunused-but-set-variable]
../GosuImpl/RubyGosu_wrap.cxx: In member function ‘virtual void SwigDirector_Window::loseFocus()’:
../GosuImpl/RubyGosu_wrap.cxx:2969:9: warning: variable ‘result’ set but not used [-Wunused-but-set-variable]
../GosuImpl/RubyGosu_wrap.cxx: In member function ‘virtual void SwigDirector_Window::releasemory()’:
../GosuImpl/RubyGosu_wrap.cxx:2976:9: warning: variable ‘result’ set but not used [-Wunused-but-set-variable]
../GosuImpl/RubyGosu_wrap.cxx: In member function ‘virtual void SwigDirector_Window::buttonDown(Gosu::Button)’:
../GosuImpl/RubyGosu_wrap.cxx:2984:9: warning: variable ‘result’ set but not used [-Wunused-but-set-variable]
../GosuImpl/RubyGosu_wrap.cxx: In member function ‘virtual void SwigDirector_Window::buttonUp(Gosu::Button)’:
../GosuImpl/RubyGosu_wrap.cxx:2998:9: warning: variable ‘result’ set but not used [-Wunused-but-set-variable]
compiling TextInputX.cpp
cc1plus: warning: command line option ‘-Wdeclaration-after-statement’ is valid for C/ObjC but not for C++ [enabled by default]
cc1plus: warning: command line option ‘-Wimplicit-function-declaration’ is valid for C/ObjC but not for C++ [enabled by default]
compiling IO.cpp
cc1plus: warning: command line option ‘-Wdeclaration-after-statement’ is valid for C/ObjC but not for C++ [enabled by default]
cc1plus: warning: command line option ‘-Wimplicit-function-declaration’ is valid for C/ObjC but not for C++ [enabled by default]
compiling Color.cpp
cc1plus: warning: command line option ‘-Wdeclaration-after-statement’ is valid for C/ObjC but not for C++ [enabled by default]
cc1plus: warning: command line option ‘-Wimplicit-function-declaration’ is valid for C/ObjC but not for C++ [enabled by default]
compiling Font.cpp
cc1plus: warning: command line option ‘-Wdeclaration-after-statement’ is valid for C/ObjC but not for C++ [enabled by default]
cc1plus: warning: command line option ‘-Wimplicit-function-declaration’ is valid for C/ObjC but not for C++ [enabled by default]
In file included from ../GosuImpl/Graphics/Font.cpp:7:0,
                 from Font.cpp:1:
../GosuImpl/Graphics/Common.hpp: In function ‘void Gosu::multiplyBitmapAlpha(Gosu::Bitmap&, Gosu::Color::Channel)’:
../GosuImpl/Graphics/Common.hpp:113:40: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
../GosuImpl/Graphics/Common.hpp:114:43: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
In file included from ../GosuImpl/Graphics/Font.cpp:8:0,
                 from Font.cpp:1:
../GosuImpl/Graphics/FormattedString.hpp: In constructor ‘Gosu::FormattedString::FormattedString(const wchar_t*, unsigned int)’:
../GosuImpl/Graphics/FormattedString.hpp:179:44: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
../GosuImpl/Graphics/FormattedString.hpp: In member function ‘std::wstring Gosu::FormattedString::unformat() const’:
../GosuImpl/Graphics/FormattedString.hpp:203:49: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
compiling Bitmap.cpp
cc1plus: warning: command line option ‘-Wdeclaration-after-statement’ is valid for C/ObjC but not for C++ [enabled by default]
cc1plus: warning: command line option ‘-Wimplicit-function-declaration’ is valid for C/ObjC but not for C++ [enabled by default]
compiling FileUnix.cpp
cc1plus: warning: command line option ‘-Wdeclaration-after-statement’ is valid for C/ObjC but not for C++ [enabled by default]
cc1plus: warning: command line option ‘-Wimplicit-function-declaration’ is valid for C/ObjC but not for C++ [enabled by default]
In file included from FileUnix.cpp:1:0:
../GosuImpl/FileUnix.cpp: In member function ‘virtual void Gosu::File::write(std::size_t, std::size_t, const void*)’:
../GosuImpl/FileUnix.cpp:99:45: warning: ignoring return value of ‘ssize_t write(int, const void*, size_t)’, declared with attribute warn_unused_result [-Wunused-result]
../GosuImpl/FileUnix.cpp: In member function ‘virtual void Gosu::File::read(std::size_t, std::size_t, void*) const’:
../GosuImpl/FileUnix.cpp:91:42: warning: ignoring return value of ‘ssize_t read(int, void*, size_t)’, declared with attribute warn_unused_result [-Wunused-result]
../GosuImpl/FileUnix.cpp: In member function ‘virtual void Gosu::File::resize(std::size_t)’:
../GosuImpl/FileUnix.cpp:76:34: warning: ignoring return value of ‘int ftruncate(int, __off_t)’, declared with attribute warn_unused_result [-Wunused-result]
compiling TextUnix.cpp
cc1plus: warning: command line option ‘-Wdeclaration-after-statement’ is valid for C/ObjC but not for C++ [enabled by default]
cc1plus: warning: command line option ‘-Wimplicit-function-declaration’ is valid for C/ObjC but not for C++ [enabled by default]
In file included from TextUnix.cpp:1:0:
../GosuImpl/Graphics/TextUnix.cpp:8:25: fatal error: SDL/SDL_ttf.h: No such file or directory
compilation terminated.
make: *** [TextUnix.o] Error 1
make failed, exit code 2
Gem files will remain installed in /home/travis/build/mrhead/ruby_arena/vendor/bundle/ruby/2.1.0/gems/gosu-0.7.50 for inspection.
Results logged to /home/travis/build/mrhead/ruby_arena/vendor/bundle/ruby/2.1.0/extensions/x86_64-linux/2.1.0/gosu-0.7.50/gem_make.out
An error occurred while installing gosu (0.7.50), and Bundler cannot continue.
Make sure that `gem install gosu -v '0.7.50'` succeeds before bundling.
...
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -3,4 +3,5 @@ rvm:
 before_install:
  - sudo apt-get update -qq
  - sudo apt-get install -qq build-essential freeglut3-dev libfreeimage-dev libgl1-mesa-dev libopenal-dev libpango1.0-dev libsdl-ttf2.0-dev libsndfile-dev libxinerama-dev
-script: rspec spec
+script:
+ - bundle exec rspec spec/

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,3 +1,6 @@
 rvm:
   - 2.1.0
+before_install:
+ - sudo apt-get update -qq
+ - sudo apt-get install -qq build-essential freeglut3-dev libfreeimage-dev libgl1-mesa-dev libopenal-dev libpango1.0-dev libsdl-ttf2.0-dev libsndfile-dev libxinerama-dev
 script: rspec spec

